spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsCounterService.java

```diff
@@ -28,15 +28,21 @@
 import org.springframework.stereotype.Service;
 
 import java.sql.Date;
-import java.time.LocalDate;
+import java.time.*;
 import java.util.StringJoiner;
+import java.util.TimeZone;
 import java.util.UUID;
 
 import eu.coatrack.api.ApiKey;
 
 @Service
 public class MetricsCounterService {
 
+    // Quick fix of #237 by setting default time zones of Admin and Gateway to UCT timezone.
+    // In the future, this system should be overhauled by the introduction of ZonedDateTime's.
+    static {
+        TimeZone.setDefault(TimeZone.getTimeZone(ZoneId.of("Europe/London")));
```

Comment:
London time is not always equal to UTC, see https://en.wikipedia.org/wiki/British_Summer_Time

So it should probably be:
```suggestion
        TimeZone.setDefault(TimeZone.getTimeZone(ZoneId.of("UTC")));
```